.circleci/config.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       py_version:
         type: string
     macos:
-      xcode: "10.0.0"
+      xcode: "12.0.0"
```

Comment:
probably need to update `cache_key_version:`, since master reuse the caching (the cache key is per branch)